db/schema.rb

```diff
@@ -16,24 +16,6 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "admins", force: :cascade do |t|
```

Comment:
Can you take a look at your code? I'm 99% sure that this migration was removed at one point but it keeps finding its way back into the codebase (or maybe we had a drop migration for it)

Either way, I'm certain this table is unnecessary. We're using the User table to store all users and using some type of mechanism for flagging certain users as admins.